src/encoding.ts

```diff
@@ -114,6 +119,12 @@ export function normalizeEncoding(encoding: Encoding, mark: Mark): Encoding {
       return normalizedEncoding;
     }
 
+    if (channel === 'tooltip') {
```

Comment:
Removed. I had this in here because normalizeEncoding is called and the normalization of the tooltip tries to add a type, but a type is irrelevant for a tooltip since it has to be a string. This is no longer logging an error with the improved valueref code, so removed.